packages/mobile/src/pincode/authentication.ts

```diff
@@ -230,3 +231,17 @@ export async function ensureCorrectPassword(
     return false
   }
 }
+
+export async function removeAccountLocally(account: string) {
+  try {
+    ValoraAnalytics.reset().catch((error) => {
+      Logger.error(TAG, 'Error attempting to reset analytics', error, true)
+    })
+    clearPasswordCaches()
+    await removeStoredItem(STORAGE_KEYS.PEPPER)
+    await removeStoredItem(passwordHashStorageKey(account))
+    // TODO: Remove the account from geth.
```

Comment:
I think you need to remove the mnemonic, but this doesn't feel like the right place to to it.
I'd recommend keeping the work in this file to *just* authentication related pieces and hoist other work to the saga that calls this.